aiida_testing/mock_code/_fixtures.py

```diff
@@ -121,7 +129,21 @@ def _get_mock_code(
             If 'generate', add new key (label) to config dictionary.
         _regenerate_test_data :
             If True, regenerate test data instead of reusing.
+
+        .. deprecated:: 0.1.0
+            Keyword `ingore_files` is deprecated and will be removed in `v1.0`. Use `ignore_paths` instead.
         """
+        if ignore_files != ('_aiidasubmit.sh', ):
+            warnings.warn(
+                'keyword `ignore_files` is deprecated and will be removed in `v1.0`. Use `ignore_paths` instead.',
+                AiidaDeprecationWarning
```

Comment:
I'm not sure if we need to use `AiidaDeprecationWarning` here - `pytest` also shows plain `DeprecationWarning` by default.
But I guess it doesn't matter much either way, so we can also keep it.

Reply:
Thanks, I can fix this in one of the next PRs.

Reply:
Done